src/getters/get-tags.ts

```diff
@@ -0,0 +1,29 @@
+import * as Unist from 'unist';
+
+import { Frontmatter } from '../frontmatter';
+
+// Priority: yaml tags
+export const getTags = (data?: Unist.Node): string[] => {
```

Comment:
```suggestion
export const getTags = (data: Unist.Parent): string[] => {
```
I'd bre more strict here. The function won't do anything is undefined is provided, so perhaps it's better to have a condition somewhere else?

Reply:
We just work with user driven data everywhere, so we're ending up having to put this check all over the place. Figure why not just do it once here?